command/user.go

```diff
@@ -284,6 +308,24 @@ func randString(n int) string {
 	return string(bs)
 }
 
+func validCredentials(usr, pwd string) (bool, string) {
+	if usr != "" {
+		if m, _ := regexp.MatchString("^[a-zA-Z0-9@._-]*$", usr); !m {
```

Comment:
Regexp returning error must be checked and processed, in case there is an error we should log it and notify somehow the user.